sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1581,15 +1585,20 @@ def __init__(self,
     super(EmbeddedGrpcWorkerHandler, self).__init__(state, provision_info,
                                                     grpc_server)
     if payload:
-      state_cache_size = payload.decode('ascii')
+      state_cache_size, data_buffer_time_limit = \
+          payload.decode('ascii').split(',')
```

Comment:
Here we're changing the payload in a backwards incompatible way. Maybe at this point we should make it a json dict?